tensorflow/lite/micro/kernels/cmsis-nn/conv.cc

```diff
@@ -159,6 +157,19 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
   output_dims.w = output->dims->data[2];
   output_dims.c = output_shape.Dims(3);
 
+  // Dynamically allocate per-channel quantization parameters.
+  // TODO(#43475): This allocation is done even for non-int8 cases to get around
+  // a bug in kernel_utils.cc which incorrectly uses per_channel_output_shift in
+  // non-int8 cases. Protect this section with a if (input->type == kTfLiteInt8)
+  // when the issue is fixed.
+  const int num_channels = filter->dims->data[kConvQuantizedDimension];
+  data->per_channel_output_multiplier =
+      reinterpret_cast<int32_t*>(context->AllocatePersistentBuffer(
```

Comment:
nit: prefer static_cast

Reply:
sure. I've fixed it for the reference implementation as well.